### PR TITLE
Experiment route: Show single experiment on page. Allow modification

### PR DESCRIPTION
**Pull request by helenmedrano**
### Changes

- Removed styling (will be updated in following PR)
Screen Shot 2021-12-14 at 10 25 39 AM

- Can now click on experiment and will redirect to a page with the single experiment. Can modify experiment name/description in this view.
Screen Shot 2021-12-14 at 10 26 24 AM

### CODE REVIEW

File: frontend/src/api.tsx [hunk 1]
```diff
@@ -6,27 +6,41 @@ import axios, { AxiosResponse } from 'axios';
...
  */
 export function createExperiment(name: string, description: string): Promise<AxiosResponse<Experiment>> {
-    const experiment: Experiment = {
-      id: 0,
-      name,
-      description,
-    };
-    let result = axios.post('/api/experiments', experiment);
-    return result;
+  const experiment: Experiment = {
```
Comment (wmedrano) [inline frontend/src/api.tsx:32]: Space changes are fine for this PR.

In separate PR, can you set the style in tsconfig too? Does this work? https://palantir.github.io/tslint/rules/indent/

File: frontend/src/utils/stringHelpers.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const stringIsEmpty = (test: string): boolean => test.trim().length === 0;
+
+export const stringsAreEqual = (string1: string, string2: string): boolean => string1.trim() === string2.trim();
```
Comment (wmedrano) [inline frontend/src/utils/stringHelpers.ts:3]: Can you add a docstring?

**Comment by wmedrano** [comment]
Nit: I think when editing, name and description should be on different lines. Seems crammed if description is long.

**Review by wmedrano — APPROVED** [review]

File: frontend/src/api.tsx (hunk 1 above)
Comment (helenmedrano) [reply frontend/src/api.tsx:32]: yeh

And will add a lint fix command to auto style the files

**Pull request merged**